graphql2/graphqlapp/contactmethod.go

```diff
@@ -32,6 +32,39 @@ func (a *ContactMethod) FormattedValue(ctx context.Context, obj *contactmethod.C
 	return formatted, nil
 }
 
+func (a *ContactMethod) LastTestMessageState(ctx context.Context, obj *contactmethod.ContactMethod) (*graphql2.NotificationState, error) {
+	t := obj.LastTestVerifyAt()
+	if t.IsZero() {
+		return nil, nil
+	}
+
+	status, _, err := a.NotificationStore.LastMessageStatus(ctx, "test_notification", obj.ID, t)
```

Comment:
Use the type defined in the `notification` package instead of hard-coding a string.

Reply:
The notification messageType doesn't match any of the message_type in outgoing_messages. I originally had notification.MessageTypeTest but got a query error so I changed it to the exact message_type string.

Reply:
Right, they are incompatible since the Go `int` does not work as a `string` that can be cast to the enum in postgres.

To make `MessageType` work with the DB driver, we need to implement Scan and Value

https://golang.org/pkg/database/sql/#Scanner
https://golang.org/pkg/database/sql/driver/#Valuer

There's a few places we've needed to do this, here's an example:
https://github.com/target/goalert/blob/master/alert/status.go

For `Value`, use a switch to return the correct string, and for `Scan` a switch from the DB string to the `MessageType`. Once those methods exist you should no longer get a query error while scanning.